drivers/adc/adc_rpi_pico.c

```diff
@@ -0,0 +1,358 @@
+/*
+ * Copyright 2021 Google LLC
+ * Copyright 2022 TOKITA Hiroshi <tokita.hiroshi@fujitsu.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT raspberrypi_pico_adc
+
+#include <drivers/adc.h>
```

Comment:
These will now need the `<zephyr/...>` prefix

Reply:
Add <zephyr/~> and reorganized includes.